src/strutil.cc

```diff
@@ -410,10 +391,8 @@ size_t FindThreeOutsideParen(StringPiece s, char c1, char c2, char c3) {
 }
 
 size_t FindEndOfLine(StringPiece s, size_t e, size_t* lf_cnt) {
-  static const char delimiters[] = "\0\n\\";
   while (e < s.size()) {
-    e += SkipUntil(s.data() + e, s.size() - e, delimiters,
-                   [](char c) { return c == 0 || c == '\n' || c == '\\'; });
+    e += strcspn(s.data() + e, "\n\\");  // skip to line end
```

Comment:
This is also a StringPiece, but it's also doing a length check, so it also shouldn't cause problems if we find a delimiter past the end of the StringPiece.